packages/engine-server/src/drivers/file/storev2.ts

```diff
@@ -67,6 +70,60 @@ import { SchemaParser } from "./schemaParser";
 import { InMemoryNoteCache } from "../../util/inMemoryNoteCache";
 import { NotesFileSystemCache } from "../../cache";
 
+export enum EngineInitErrorType {
+  DUPLICATE_NOTE_ID = "duplicate note id",
+}
+
+export class DuplicateNoteError extends BaseDendronError<EngineInitErrorType.DUPLICATE_NOTE_ID> {
```

Comment:
nit: my pref would be to have this class in a separate file, but I don't feel too strongly about it. 

Could you also add a small class header?